src/main/kotlin/com/tschuchort/compiletesting/KotlinCompilation.kt

```diff
@@ -326,6 +331,8 @@ class KotlinCompilation {
 		it.destination = classesDir.absolutePath
 		it.classpath = commonClasspaths().joinToString(separator = File.pathSeparator)
 
+		it.pluginClasspaths = pluginClasspaths.map(File::getAbsolutePath).toTypedArray()
```

Comment:
The `commonK2JVMArgs` will be used in both steps: `stubsAndApt` and `compileKotlin`. So given compiler plugins will be run both times. Is this the desired behaviour? Unless the kapt plugin prevents the compiler from continuing with other plugins, like it short circuits the regular compilation. @Foso may know more.

Reply:
Oh, I didn't have that problem because I didn't provide `annotationProcessors`, so `stubsAndApt` finished in the first conditional. Let's see the opinion by @Foso.

Reply:
This issue still deserves some attention. Are we really sure that compiler plugins should be executed in both steps?